specs/Extensions.md

```diff
@@ -0,0 +1,585 @@
+                    {
+                        wil::com_ptr<ICoreWebView2BrowserExtension> extension;
+                        extensions->GetValueAtIndex(index, &extension);
+
+                        wil::unique_cotaskmem_string id;
+                        wil::unique_cotaskmem_string name;
+
+                        extension->get_Id(&id);
+                        if (_wcsicmp(id.get(), dialog.input.c_str()) == 0)
+                        {
+                            extension->Remove(
+                                Callback<ICoreWebView2BrowserExtensionRemoveCompletedHandler>(
+                                    [](HRESULT error) -> HRESULT
+                                    {
+                                        if (error != S_OK)
+                                        {
+                                            ShowFailure(error, L"Failed to toggle extension enabled.");
+                                        }
+                                        MessageBox(
+                                            nullptr, L"Done", L"Remove Extension", MB_OK);
+                                        return S_OK;
+                                    })
+                                    .Get());
+                        }
+                    }
+                }
+                return S_OK;
+            })
+            .Get());
+}
+
+void BrowserExtensionService::ToggleExtensionEnabled()
+{
+    // Get the profile object.
+    auto webView2_13 = m_webView.try_query<ICoreWebView2_13>();
+    wil::com_ptr<ICoreWebView2Profile> webView2Profile;
+    CHECK_FAILURE(webView2_13->get_Profile(&webView2Profile));
+    auto profile6 = webView2Profile.try_query<ICoreWebView2Profile6>();
+
+    profile6->GetExtensions(
+        Callback<ICoreWebView2ProfileGetBrowserExtensionCompletedHandler>(
+            [this](HRESULT error, ICoreWebView2BrowserExtensionList* extensions) -> HRESULT
+            {
+                TextInputDialog dialog(
+                    m_appWindow->GetMainWindow(),
+                    remove ? L"Remove Extension" : L"Disable/Enable Extension",
+                    L"Extension ID:", extensionIdString.c_str());
+                if (dialog.confirmed)
+                {
+                    for (UINT index = 0; index < extensionsCount; ++index)
+                    {
+                        wil::com_ptr<ICoreWebView2BrowserExtension> extension;
+                        extensions->GetValueAtIndex(index, &extension);
+
+                        wil::unique_cotaskmem_string id;
+                        wil::unique_cotaskmem_string name;
+
+                        extension->get_Id(&id);
+                        if (_wcsicmp(id.get(), dialog.input.c_str()) == 0)
+                        {
+                            BOOL enabled = FALSE;
+                            extension->get_IsEnabled(&enabled);
+
+                            extension->SetEnabled(
```

Comment:
I think `Enable(bool, callback)` is acceptable here. This is a case where the meaning of a Boolean parameter is obvious from the method name.